fspiop-api/documents/Tracing v1.0.md

```diff
@@ -0,0 +1,895 @@
+            "partyIdentifier": "SE4550000000058398257466",
+            "fspId": "BankNrOne"
+        }
+    },
+    "amountType": "RECEIVE",
+    "amount": {
+        "amount": "100",
+        "currency": "USD"
+    },
+    "transactionType": {
+        "scenario": "TRANSFER",
+        "initiator": "PAYER",
+        "initiatorType": "CONSUMER"
+    },
+    "note": "From Mats",
+    "expiration": "2017-11-15T22:17:28.985-01:00"
+}
+```
+
+4.1.2.2. Request Quote from Switch to MobileMoney
+
+Switch receives the request, and mutates the trace information before forwarding the request to MobileMoney.
+
+**Note:** See previous section [4.2.1. Request Party Information](#421-request-party-information) for more detail.
+
+The following trace headers will be include in the request:
+
+**Note:** The fields highlighted below as **`value`** will indicate a newly generated or modified value depending on context. Non-highlighted fields will indicate an unchanged value.
+
+> traceparent: 00-91e502e28cd723686e9940bd3f378f85-**`379cc3faa566cf90`**-01<br/>
+> tracestate: **`moja`**=**`379cc3faa566cf90`**,banknrone=0c3e9f44f921ca06,mobilemoney=619d4c9d431ca708<br/>
+
+<br/>
+The request sample including trace headers:
+
+```http
+POST /quotes HTTP/1.1
+Accept: application/vnd.interoperability.quotes+json;version=1
+Content-Type: application/vnd.interoperability.quotes+json;version=1.0
+Content-Length: 975
+Date: Tue, 15 Nov 2017 10:13:40 GMT
+FSPIOP-Source: BankNrOne
+FSPIOP-Destination: MobileMoney
+traceparent: 00-91e502e28cd723686e9940bd3f378f85-379cc3faa566cf90-01
+tracestate: moja=379cc3faa566cf90,banknrone=0c3e9f44f921ca06,mobilemoney=619d4c9d431ca708
+{
+    "quoteId": "7c23e80c-d078-4077-8263-2c047876fcf6",
+    "transactionId": "85feac2f-39b2-491b-817e-4a03203d4f14",
+    "payee": {
+        "partyIdInfo": {
+            "partyIdType": "MSISDN",
+            "partyIdentifier": "123456789",
+            "fspId": "MobileMoney"
+        }
+    },
+    "payer": {
+        "personalInfo": {
+            "complexName": {
+                "firstName": "Mats",
+                "lastName": "Hagman"
+            }
+        },
+        "partyIdInfo": {
+            "partyIdType": "IBAN",
+            "partyIdentifier": "SE4550000000058398257466",
+            "fspId": "BankNrOne"
+        }
+    },
+    "amountType": "RECEIVE",
+    "amount": {
+        "amount": "100",
+        "currency": "USD"
+    },
+    "transactionType": {
+        "scenario": "TRANSFER",
+        "initiator": "PAYER",
+        "initiatorType": "CONSUMER"
+    },
+    "note": "From Mats",
+    "expiration": "2017-11-15T22:17:28.985-01:00"
+}
+```
+
+4.1.2.3. Quote response callback from MobileMoney to Switch
+
+MobileMoney receives the request, and mutates the trace information before responding with the **PUT /quotes** callback to the Switch.
+
+**Note:** See previous section [4.2.1. Request Party Information](#421-request-party-information) for more detail.
```

Comment:
The link seems to be incorrect, I would guess that it should be to 4.1.1.2. Request Party Information from Switch to MobileMoney?

Reply:
Fixed. It's meant to refer to "[4.1.1. Request Party Information](#411-request-party-information)" in general.